scripts/ci_test.sh

```
@@ -5,42 +5,54 @@ set -o errexit
 set -o pipefail
 
 SECRETS_BASE_PATH="${SECRETS_BASE_PATH:-/var/run/secrets}"
+JOB_NAME="assisted-chat-eval-test"
 
-oc create secret generic -n "$NAMESPACE" assisted-chat-ssl-ci --from-file=client_id=/var/run/secrets/sso-ci/client_id \
-    --from-file=client_secret=/var/run/secrets/sso-ci/client_secret
+if ! oc get secret -n "$NAMESPACE" assisted-chat-ssl-ci &>/dev/null; then
```

Shellcheck (0.10.0)

[error] Couldn't find 'fi' for this 'if'.

(SC1046)